model/permission.go

```diff
@@ -695,6 +698,18 @@ func initializePermissions() {
 		"authentication.permissions.use_group_mentions.description",
 		PermissionScopeSystem,
 	}
+	PERMISSION_SYSCONSOLE_READ_BILLING = &Permission{
+		"sysconsole_read_billing",
+		"authentication.permissions.use_group_mentions.name",
+		"authentication.permissions.use_group_mentions.description",
+		PermissionScopeSystem,
+	}
+	PERMISSION_SYSCONSOLE_WRITE_BILLING = &Permission{
+		"sysconsole_write_billing",
+		"authentication.permissions.use_group_mentions.name",
+		"authentication.permissions.use_group_mentions.description",
```

Comment:
Should the "use_group_mentions.name" and "use_group_mentions.description" be changed

Reply:
To be honest, I'm not sure. If it does need to be changed, what should it be changed too?

Reply:
I'm not familiar with how it is used, please investigate. The names just didn't look right to me.

Reply:
Done. As discussed, passing empty string.